### ISSUE TITLE
Checkbox (Walnut)

### ISSUE THREAD

**Issue by jrob24** [vmware/webcommander#145]
I was able to modify the genDefJson.ps1 and ui-min.js so that I could use the checkbox type. However I would like to be able to pass a value to the powershell script when the checkbox is checked. I can't figure out how to do this. Still trying to learn JS. Looks like I should be able to do it with the value property. 

http://www.w3schools.com/jsref/prop_checkbox_value.asp

I can't quite figure out how to add to ui-min.js. Any suggestions? Here are my addtions:

![image](https://cloud.githubusercontent.com/assets/12749455/13586536/2eb35f84-e487-11e5-88e6-6a421524bf21.png)

![image](https://cloud.githubusercontent.com/assets/12749455/13586539/37c8304a-e487-11e5-9913-3dc3d582e33c.png)

**Comment by 9whirls** [comment]
Right, you need to add js function to tell whether the checkbox is "checked". So I prefer using dropdown list.
